omod/src/main/webapp/resources/css/openmrs.css

```diff
@@ -1,4 +1,9 @@
+/*Assigned Font to Html Instead Of * To Fix Issue Where Other Fonts May Get Overriden
```

Comment:
Can you remove the comments. We don't need those. They should ideally go in your commit or PR description.